src/main/java/com/glitchcog/fontificator/bot/Message.java

```diff
@@ -525,11 +513,7 @@ else if (emojiConfig.isTwitchBadgesEnabled() && (testBadge = twitchBadgeBank.get
         // Configured for no emoji, so just chars
         else
         {
-            String casedContent = applyCasing(content, messageConfig.getMessageCasing());
```

Comment:
This is the case that didn't handle codepoints still when emotes were turned off.